tests/cpp/aot/python_scripts/graph_aot_test.py

```diff
@@ -11,17 +11,17 @@ def compile_graph_aot(arch):
         return
 
     @ti.kernel
-    def run0(base: int, arr: ti.types.ndarray(field_dim=1, dtype=ti.i32)):
+    def run0(base: int, arr: ti.types.ndarray(ndim=1, dtype=ti.i32)):
```

Comment:
nit: shall we reorder this to dtype first given they're both keywords?

Reply:
There are lots of such cases in examples and tests. Even `any_arr` is still present in our codebase. Let's do a full sweep in the next PR.